docs/loadflow/parameters.md

```diff
@@ -213,58 +213,67 @@ This parameter 'slows down' the Newton-Raphson by scaling the state vector betwe
+**newtonKrylovLineSearch**    
 Only applies if **acSolverType** is `NEWTON_KRYLOV`.
 Activates or deactivates line search for the Newton-Raphson Kinsol solver.
 The default value is `false`.
 
-**plausibleActivePowerLimit**  
+**plausibleActivePowerLimit**    
 The `plausibleActivePowerLimit` property is an optional property that defines a maximal active power limit for generators to be considered as participating elements for:
 - slack distribution (if `balanceType` equals to any of the `PROPORTIONAL_TO_GENERATION_<any>` types)
 - slack selection (if `slackBusSelectionMode` equals to `LARGEST_GENERATOR`)
 
 The default value is $5000 MW$.
 
-**minPlausibleTargetVoltage** and **maxPlausibleTargetVoltage**  
+**minPlausibleTargetVoltage** and **maxPlausibleTargetVoltage**    
 Equipments with voltage regulation target voltage outside these per-unit thresholds
 are considered suspect and are discarded from regulation prior to load flow resolution.  
 The default values are `0.8` and `1.2` and they must be greater or equal to `0`.
 
-**minRealisticVoltage** and **maxRealisticVoltage**  
+**minRealisticVoltage** and **maxRealisticVoltage**    
 These parameters are used to identify if Newton-Raphson has converged to an unrealistic state.
 For any component where a bus voltage is solved outside these per-unit thresholds, the component solution is deemed unrealistic
 and its solution status is flagged as failed.  
 The default values are `0.5` and `1.5` and they must be greater or equal to `0`.
 
-**reactiveRangeCheckMode**  
-This parameter defines how to check the reactive limits $MinQ$ and $MaxQ$ of a generator. If the range is too small, the generator is discarded from voltage control.
-- `MIN_MAX` mode checks if the reactive range at $MaxP$ is above a threshold and if the reactive range at $MinP$ is not zero.
-- `MAX` mode if the reactive range at $MaxP$ is above a threshold.
-- `TARGET_P` if the reactive range at $TargetP$ is above a threshold
+**reactiveRangeCheckMode**    
+OpenLoadFlow discards voltage control for generators with a too small reactive power range, because in practice a too
+small reactive power ranger means limited to zero voltage control capability.
+
+For a given active power output, the reactive power range is defined as $MaxQ - MinQ$ (always a positive value).  
+The *maximum* and *minimum* reactive range of a generator is:
+- for generators without reactive limits: infinity 
+- for generators with reactive limits defined by a pair of [min/max values](inv:powsyblcore:*:*:#min-max-reactive-limits), both minimum and maximum reactive range are equal to $MaxQ - MinQ$ 
+- for generators with reactive limits defined by a [reactive capability curve](inv:powsyblcore:*:*:#reactive-capability-curve), the minimum (resp. maximum) reactive range is obtained by finding the curve point having the minimum (resp. maximum) $MaxQ - MinQ$.
+
+The `reactiveRangeCheckMode` parameter defines how generator reactive power range is to be tested:

```

Comment:
Maybe you should add that if the test fails, the generator cannot control voltage. For me it is not clear.

Reply:
I agree, fixed